README.md

```diff
@@ -1 +1,59 @@
-# modules
\ No newline at end of file
+# Modules
+
+Site made with JavaScrip:
+
+- Objects.
+- Classes,
+- SPA Dinamic.
+- Modules Jnodes.
+
+## Built With:
+
+- HTML.
+- CSS.
+- JavaScript
+- Luxon
+- Check the code with Linters tests:
+  _Webhint_
+  _Stylelint_
+  _Lighthouse_
+  _esLint_
+
+## Live Demo
+
+[Live Demo](https://zkr024.github.io/modules/)
```

Comment:
- [ ] Kindly update your live link.

Reply:
thank you @Deepakdanger for the feedback.
I'll work on it...but I have a question. It works from the vs code go live option, but if I try to open the index.html file, it prompts this error:  
'null' has been blocked by CORS policy: Cross origin requests are only supported for protocol schemes: http, data, chrome, chrome-extension, chrome-untrusted, https. 
Any suggestion there maybe